### PR TITLE
feat: 2.13.3 - filter shipping, new google SSO and add bank logo coupon

### PR DESCRIPTION
**Pull request by DiebrataA**
SSO (Notes: Tampilan Logo google yang ada background putih sudah disetujui designer ^^): 
register_SSO
functinality_SSO
login_SSO

Coupon Cart:
coupon_cart

Coupon PlaceORder:
coupon_placeOrder

Coupon Payment Method:
coupon_paymentMethod

Coupon Loading:
coupon_loading

Coupon ShippingMethod:
coupon_shippingMethod

MoreShipipng:
moreShipping

Footer Expanded:
Footer_expanded

### CODE REVIEW

**Review by sultanfariz — APPROVED** [review]

File: public/scss/pages/Login.module.scss [hunk 1]
```diff
@@ -105,6 +105,19 @@
...
 
+    & > div 
+    {
+      @include flex(row, center, center);
+      margin: 0 12px;
+      border-radius: 50px; 
+      box-shadow: none;
+      background: $color_rose_bg2; 
+      width: 54px;
+      height: 54px !important;
```
Comment (mayun19) [inline public/scss/pages/Login.module.scss:116]: ini bisa pakai mixin kah ?

**Review by mayun19 — CHANGES_REQUESTED** [review]

File: public/scss/components/Shipping.module.scss [hunk 2]
```diff
@@ -163,4 +163,11 @@
       }
     }
   }
+
+  &_moreShipping
+  {
+    cursor: pointer;
+    padding: 16px 16px;
```
Comment (inezamandha) [inline public/scss/components/Shipping.module.scss:170]: Bisa di 16px kan saja sepertinyaa

File: public/scss/components/Shipping.module.scss [hunk 3]
```diff
@@ -163,4 +163,11 @@
...
+    border: 1px solid $color_border
```
Comment (inezamandha) [inline public/scss/components/Shipping.module.scss:171]: Boleh kasih ; di belakangnyaa biar seragam?

**Review by inezamandha — CHANGES_REQUESTED** [review]

**Review by mayun19 — APPROVED** [review]

File: pages/[lng]/shipping_method/index.tsx [hunk 4]
```diff
@@ -56,6 +56,7 @@ const classesShippingMethod = {
   shippingPriceDivClass: styleShipping.shipping_item__price,
   pinPointLocationClassName: `${styleBtn.btn} ${styleBtn.btn_secondary} ${styleShipping.shipping_item__pinButton} mb-3`,
   shippingErrorMsgClass: styleShipping.shipping_item__errorMsg,
+  moreShippingClassName:styleShipping.shipping_moreShipping,
```
Comment (inezamandha) [inline pages/[lng]/shipping_method/index.tsx:59]: kasih spasi boleh sebelah : ?

**Review by inezamandha — APPROVED** [review]

**Pull request merged**